cmake/onnxruntime_pyop.cmake

```diff
@@ -1,30 +1,45 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License.
 
-FIND_PACKAGE(PythonLibs)
```

Comment:
can you add a one line comment before each of the exec_programs saying what it is intending to achieve?

Reply:
Add comments.